### ISSUE TITLE
Blank page when opening repository after updates

### ISSUE THREAD

**Issue by r00ntje** [projectfork/Projectfork#1717]
Before updating to the latest Joomla and Projectfork version, I made an Akeeba backup. Updated.  Afterwards, opening the users or repository pages results in a blank page. 

With error reporting set to the maximum, the page is still blank. No error is shown..

What is even worse, the backup I made has magically disappeared and I can't roll back to a previous version. 

I tried uninstalling Projectfork and installing an older version, but the problem persists.

**Comment by angek** [comment]
have you checked in /administrator/components/com_akeeba/backup for the .jpa file?
I've used Akeeba for quite some time and have never experienced such an issue.

**Comment by r00ntje** [comment]
Hi, angek. Yeah, I followed the directions of Akeeba backup manager to that location. There were no backups in that folder. But when making a backup, it does end up in that folder. After a while, it seemed to have disappeared by itself. 

Anyway, I completely rebuild the website on a another domain, using only the Akeeba extracted database file. It was actually a lot less work than I expected. I had now a fresh website, same database and all the components and template installed again. Everything worked again as it should. including Projectfork. I transferred the website over the old faulty one with Akeeba without any problems.

Everything seems to be working now, which leads me to believe that it wasn't a problem with Projectfork. My website was corrupted somehow, which may have lead to Akeeba backups disappearing. On the new site, I have not seen this happen again.

One other thing, perhaps it was that I reached my website size quota? Therefore, It could not save the file properly?
